src/Network/Network/help/Set-AzFirewallPolicyRuleCollectionGroup.md

```diff
@@ -0,0 +1,172 @@
+---
+external help file: Microsoft.Azure.PowerShell.Cmdlets.Network.dll-Help.xml
+Module Name: Az.Network
+online version: https://docs.microsoft.com/en-us/powershell/module/az.network/set-azfirewallpolicyrulecollectiongroup
+schema: 2.0.0
+---
+
+# Set-AzFirewallPolicyRuleCollectionGroup
+
+## SYNOPSIS
+{{ Fill in the Synopsis }}
+
+## SYNTAX
+
+```
+Set-AzFirewallPolicyRuleCollectionGroup -Name <String> -Priority <UInt32>
+ -RuleCollection <PSAzureFirewallPolicyBaseRuleCollection[]> -ResourceGroupName <String>
+ -AzureFirewallPolicy <PSAzureFirewallPolicy> [-DefaultProfile <IAzureContextContainer>] [-WhatIf] [-Confirm]
+ [<CommonParameters>]
+```
```

Comment:
This cmdlet is completely different from the design. Would you mind explaining why?

Reply:
Fixed.